python/ipyleaflet_core/ipyleaflet/leaflet.py

```diff
@@ -2215,6 +2192,135 @@ def clear_markers(self):
         self.send({"msg": "clear_markers"})
 
 
+class DrawControl(DrawControlBase):
+    """DrawControl class.
+
+    Drawing tools for drawing on the map.
+    """
+
+    _view_name = Unicode("LeafletDrawControlView").tag(sync=True)
+    _model_name = Unicode("LeafletDrawControlModel").tag(sync=True)
+
+    # Enable each of the following drawing by giving them a non empty dict of options
+    # You can add Leaflet style options in the shapeOptions sub-dict
+    # See https://github.com/Leaflet/Leaflet.draw#polylineoptions and
+    # https://github.com/Leaflet/Leaflet.draw#polygonoptions
+    circlemarker = Dict({"shapeOptions": {}}).tag(sync=True)
+
+    last_draw = Dict({"type": "Feature", "geometry": None})
+    last_action = Unicode()
+
+    def __init__(self, **kwargs):
+        super(DrawControl, self).__init__(**kwargs)
+        self.on_msg(self._handle_leaflet_event)
+
+    def _handle_leaflet_event(self, _, content, buffers):
+        if content.get("event", "").startswith("draw"):
+            event, action = content.get("event").split(":")
+            self.last_draw = content.get("geo_json")
+            self.last_action = action
+            self._draw_callbacks(self, action=action, geo_json=self.last_draw)
+
+
+class GeomanDrawControl(DrawControlBase):
+    """GeomanDrawControl class.
+
+    Alternative drawing tools for drawing on the map provided by Leaflet-Geoman.
+    """
+
+    _view_name = Unicode("LeafletGeomanDrawControlView").tag(sync=True)
+    _model_name = Unicode("LeafletGeomanDrawControlModel").tag(sync=True)
+
+    # Current mode & shape
+    # valid values are: 'draw', 'edit', 'drag', 'remove', 'cut', 'rotate'
+    # for drawing, the tool can be added after ':' e.g. 'draw:Marker'
```

Comment:
```suggestion
    # for drawing, the tool can be added after ':' e.g. 'draw:marker'
```
not sure if that should match the trait name